CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+# How to Contribute
```

Comment:
Could we follow the pattern of adding a link to our main contributing guidelines at the top?